server/lib/tuist/authentication.ex

```diff
@@ -84,7 +84,7 @@ defmodule Tuist.Authentication do
   def encode_and_sign(resource, claims \\ %{}, opts \\ []) do
     projects =
       resource
-      |> Projects.get_all_project_accounts()
+      |> Projects.get_all_project_accounts(recent: 5)
```

Comment:
what do you think about:
- moving away from `get_all_project_accounts` here since we need just the full handle
- and instead leveraging preloads to get access to the account name

If we do that, we can have a method such as `list_all_accessible_projects(subject)` which would run a regular `Flop` and then we can leverage the usual listing capabilities to get a limited set of projects.